.travis.yml

```diff
@@ -2,16 +2,18 @@ language: php
 
 dist: precise
 
-php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
-    - 7.2
-    - 7.3
-    - nightly
+jobs:
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
+    - php: 7.2
+    - php: 7.3
+      env: WITH_COVERAGE=true
```

Comment:
We probably do not need to collect coverage on all builds - running on the latest version of PHP increases the likelihood that it's a fast run nonetheless.